src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-architecture-recipes.mdx

```diff
@@ -48,7 +48,7 @@ If you set up the collector as a sidecar, OpenTelemetry calls this an agent:
 
 ## Tail-based sampling [#tail-based]
 
-By default, OpenTelemetry uses head-based sampling, which means traces are randomly sampled up front before details about the completed traces are known. As an alternative, you can configure OpenTelemetry to use our tail-based sampling, which analyzes all your traces after they're completed. If you want to use tail-based sampling, set up the collector to send every span to a trace observer in New Relic Edge where they're analyzed.
+By default, OpenTelemetry uses head-based sampling, which means traces are randomly sampled up front before details about the completed traces are known. As an alternative, you can configure OpenTelemetry to use our tail-based sampling, which analyzes all your traces after they're completed. If you want to use tail-based sampling, set up the collector to send every span to a trace observer in New Relic Edge where they're analyzed. For more information, see [Tail Sampling Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor).
```

Comment:
I think this needs to be a separate option. There are 2 options here getting shmoosed into 1 😄 

Options:
1. Use infinite tracing tail-based sampling
2. Use OpenTelemetry-Collector based tail-based sampling

Reply:
Thanks for the feedback, @a-feld. I've separated these options for tail-based sampling.